src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
+import core.basesyntax.lottery.Ball;
+import core.basesyntax.lottery.Color;
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
`Random` must be a class field
```suggestion
        int index = random.nextInt(Color.values().length);
```